src/main/java/http/helper/Json.java

```diff
@@ -0,0 +1,44 @@
+package http.helper;
```

Comment:
This is a lot of helpers - they are all untested and not used? Are they useful for you now?

Reply:
Thank you. i've removed this code now.